share/zephyrbuild-package/cmake/ZephyrBuildConfig.cmake

```
@@ -1,3 +1,4 @@
 # The ZephyrBuildConfig is simply including NCS package.
+# The ZephyrBuildConfig is simply including NCS package.
 # The NCS package can be used to precisely identify an NCS installation, in case there are multiple Zephyr derivatives installed.
-include(${CMAKE_CURRENT_LIST_DIR}/../../ncs-package/cmake/NcsConfig.cmake)
+include(${CMAKE_CURRENT_LIST_DIR}/../../ncs-package/cmake/NcsConfig)
```

- The additional comment line is redundant and could be removed for clarity.  
```suggestion
-# The ZephyrBuildConfig is simply including NCS package.
```
- The file extension `.cmake` is missing in the `include` statement, which may cause issues in finding the correct file.  
```suggestion
+include(${CMAKE_CURRENT_LIST_DIR}/../../ncs-package/cmake/NcsConfig.cmake)
```